test/harden/lodash_template.js

```diff
@@ -11,6 +11,7 @@ const _ = require('lodash');
 // eslint-disable-next-line no-restricted-modules
 const template = require('lodash/template');
 const fp = require('lodash/fp');
+// eslint-disable-next-line no-restricted-modules
```

Comment:
@legrego Turns out we actually have some code to harden `lodash.template` in `src/setup_node_env/harden/lodash_template.js`. Do we still want to get disallow the use of `template` in ESLint?

Reply:
> Turns out we actually have some code to harden `lodash.template` in `src/setup_node_env/harden/lodash_template.js`. Do we still want to get disallow the use of `template` in ESLint?

@watson Yes I think we do. We hardened `lodash.template` because it used to be (and might still be) used transitively as part of the Kibana server's logging framework, and this was our only option to harden that code path.

I don't think we should permit additional usages -- if this becomes an issue for downstream teams, then we can reevaluate. For the time being, I think it makes sense to lock this down.